src/core/accumulators/MeanVarianceCalculator.hpp

```diff
@@ -40,7 +40,7 @@ class MeanVarianceCalculator : public AccumulatorBase {
      via the interface. */
   std::string get_internal_state() const;
   void set_internal_state(std::string const &);
-  std::vector<std::size_t> shape() const { return m_obs->shape(); }
```

Comment:
why did you remove the namespace?

Reply:
side-effect of copy-pasting the method from another class... I remember there have been issues in CI when source files did not include a standard header that provides the C-version `size_t`. Can the `std::size_t` version be enforced in CI? Clang-tidy doesn't seem to have a rule for it.

Reply:
i checked, it actually doesn't seem to make a difference